### PR TITLE
LLM: add README.md for Long-Context examples.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description

This PR is to add README.md for Long-Context examples.

### CODE REVIEW

File: python/llm/example/GPU/Long-Context/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Running Long-Context generation using IPEX-LLM on Intel GPU
+
+Long-Context Generation is a critical aspect in various applications, such as document summarization, extended conversation handling, and complex question answering. Effective long-context generation can lead to more coherent and contextually relevant responses, enhancing user experience and model utility.
+
+This folder contains examples of running long-context generation with IPEX-LLM on Intel GPU:
```
Comment (jason-dai) [inline python/llm/example/GPU/Long-Context/README.md:5]: which GPU?

**Comment by jason-dai** [comment]
Also need to update `python/llm/example/GPU/README.md`

File: python/llm/example/GPU/Long-Context/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<!-- Maybe like this after adding more examples:
+- [Single GPU](Single GPU): single GPU examples w & w/o batch.
+- [Multiple GPU](Multiple GPU): multiple GPU examples w & w/o batch. -->
+- [LLaMA2-32K](LLaMA2-32K): examples of running LLaMA2-32K models with single GPU and without batch.
+- [ChatGLM3-32K](Chatglm3-32K): examples of running ChatGLM3-32K models with single GPU and without batch.
```
Comment (hkvision) [inline python/llm/example/GPU/Long-Context/README.md:11]: may remove single GPU and without batch?

File: python/llm/example/GPU/Long-Context/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Maximum Input Length for Different Models with INT4/FP8 Precision.
+
+- **INT4**
+
+    | Model Name | Low Memory Mode | Maximum Input Length |
```
Comment (hkvision) [inline python/llm/example/GPU/Long-Context/README.md:17]: IPEX_LLM_LOW_MEM and 1/0 in the table?

File: python/llm/example/GPU/Long-Context/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    | -- | -- | -- |
+    | LLaMA2-7B-32K | Disable | 10K |
+    |  | Enable | 10K(Need to verify) |
```
Comment (hkvision) [inline python/llm/example/GPU/Long-Context/README.md:20]: quick verify this before merge?

File: python/llm/example/GPU/Long-Context/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+This folder contains examples of running long-context generation with IPEX-LLM on Intel GPU:
+
+<!-- Maybe like this after adding more examples:
+- [Single GPU](Single GPU): single GPU examples w & w/o batch.
+- [Multiple GPU](Multiple GPU): multiple GPU examples w & w/o batch. -->
+- [LLaMA2-32K](LLaMA2-32K): examples of running LLaMA2-32K models with single GPU and without batch.
+- [ChatGLM3-32K](Chatglm3-32K): examples of running ChatGLM3-32K models with single GPU and without batch.
+
+### Maximum Input Length for Different Models with INT4/FP8 Precision.
```
Comment (hkvision) [inline python/llm/example/GPU/Long-Context/README.md:13]: on Arc770 (16G GPU memory)?

**Comment by jason-dai** [comment]
Also need to update `python/llm/example/GPU/README.md`

File: python/llm/example/GPU/Long-Context/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+- **FP8**
+
+    | Model Name | Low Memory Mode | Maximum Input Length | Output Length |
+    | -- | -- | -- | -- |
+    | LLaMA2-7B-32K | Disable | 7K | 512 |
+    |  | Enable | 9K | 512 |
+    | ChatGLM3-6B-32K | Disable | 8K | 512|
+    |  | Enable | 9K | 512 |
+
+> Note: If you need to run longer input or use less memory, please set `IPEX_LLM_LOW_MEM=1` to enable **low memory mode**, which will enable memory optimization and may slightly affect the latency performance.
```
Comment (hkvision) [inline python/llm/example/GPU/Long-Context/README.md:33]: remove latency, just performance?

File: python/llm/example/GPU/README.md [hunk 7]
```diff
@@ -11,6 +11,7 @@ This folder contains examples of running IPEX-LLM on Intel GPU:
 - [PyTorch-Models](PyTorch-Models): running any PyTorch model on IPEX-LLM (with "one-line code change")
 - [Speculative-Decoding](Speculative-Decoding): running any ***Hugging Face Transformers*** model with ***self-speculative decoding*** on Intel GPUs
 - [ModelScope-Models](ModelScope-Models): running ***ModelScope*** model with IPEX-LLM on Intel GPUs
+- [Long-Context](Long-Context): running **long-context** generation with IPEX-LLM on Intel Arc™ A770 Graphics.
```
Comment (hkvision) [inline python/llm/example/GPU/README.md:14]: just on Intel GPUs?

**Review by hkvision — APPROVED** [review]

File: python/llm/example/GPU/README.md (hunk 7 above)
Comment (lalalapotter) [reply python/llm/example/GPU/README.md:14]: keep arc a770 for now, maybe change to intel gpus after tested on other gpus.

**Pull request merged**